connectors/aibuilder/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM aiod_metadata_catalogue
+
+COPY cron /etc/cron.d/aiod
+COPY mlmodels.sh /opt/connectors/script/mlmodels.sh
+COPY entry.sh /opt/connectors/script/entry.sh
+
+USER root
```

```suggestion
USER root
RUN apt-get update
```

cron was not installable for me otherwise